alphalens/performance.py

```diff
@@ -453,9 +453,11 @@ def average_cumulative_return_by_quantile(quantized_factor, forward_returns,
     ----------
     quantized_factor : pd.Series
         Factor quantiles indexed by date and asset.
-    forward_returns : pd.Series - MultiIndex
-        Daily forward returns indexed by date and asset and
-        optional a custom group.
+    prices : pd.DataFrame
```

Comment:
I didn't want to rely on forward returns because of issue #102 and also because it's a bad assumption thinking we have 1 period forward return for every day (while it is a requirement to have daily pricing data that spans the factor analysis time period). So I changed average_cumulative_return_by_quantile to use pricing data instead of forward_returns_period_1